ext/mysqli/mysqli_prop.c

```diff
@@ -158,23 +179,21 @@ static int link_affected_rows_read(mysqli_object *obj, zval *retval, bool quiet)
 
  	mysql = (MY_MYSQL *)((MYSQLI_RESOURCE *)(obj->ptr))->ptr;
 
-	if (!mysql) {
-		ZVAL_NULL(retval);
-	} else {
-		CHECK_STATUS(MYSQLI_STATUS_VALID, quiet);
+	ZEND_ASSERT(mysql);
+	CHECK_STATUS(MYSQLI_STATUS_VALID, quiet);
```

Comment:
`CHECK_STATUS()` seems to be duplicated. Can we just use `MYSQLI_STATUS_VALID ` instead of `MYSQLI_STATUS_INITIALIZED` a few lines above, and remove this invocation?

Reply:
Yes, we can. Please make it the same as `stmt_affected_rows_read`. You also have an orphaned `}` on line 199